src/com/comeet/Room.java

```diff
@@ -161,7 +161,7 @@ public void addFreeBusyTime(FreeBusySlot slot) {
     }
 
     @XmlElement(name = FREEBUSY)
-    public List<FreeBusySlot> getFreeBusy() {

```

Comment:
I don't think you need to change the class member names.  The XmlElement name parameters should have fixed the generated json.